src/view/com/lightbox/ImageViewing/components/ImageItem/ImageItem.android.tsx

```diff
@@ -43,13 +46,14 @@ type Props = {
   onZoom: (isZoomed: boolean) => void
   isScrollViewBeingDragged: boolean
   showControls: boolean
+  dismissSwipePan: PanGesture
 }
 const ImageItem = ({
   imageSrc,
   onTap,
   onZoom,
-  onRequestClose,
   isScrollViewBeingDragged,
+  dismissSwipePan,
```

Comment:
we're moving the "swipe to dismiss" handling upwards because it needs to integrate with the opening animation which isn't forked by platform